specs/latest/2.0/index.html

```diff
@@ -967,7 +967,7 @@ <h3><a name="WEBGLRENDERINGCONTEXT">The WebGL context</a></h3>
   [WebGLHandlesContextLoss] GLboolean isVertexArray(WebGLVertexArrayObject? vertexArray);
   void bindVertexArray(WebGLVertexArrayObject? array);
 };
-WebGL2RenderingContextBase includes WebGLRenderingContextBase;
+WebGL2RenderingContext includes WebGLRenderingContextBase;
```

Comment:
There's nothing that prevents you from using an includes statement before you actually declare the interface that includes it, but it looks weird.

Reply:
I suggest you move it right above the line 975 (`WebGL2RenderingContext includes WebGL2RenderingContextBase;`).

Reply:
Ah, nice review. Done.
Thank you!